makewheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+MY_MODULES=$(git submodule status | awk '{print $2}')
+
+
+# zipfile=$PWD/modules.zip
+
+# rm -f $zipfile
+
+for m in $MY_MODULES; do
+    cd $m
+    python3 setup.py bdist_wheel
+    mv dist/*.whl ..
+    cd ..
```

Comment:
Can we just make it that the default install makes wheels?

Reply:
Hmm... maybe not. We want `pip install --editable`. Maybe we can have two modes of the installer?